src/components/react/SyntaxHighlighter.tsx

```diff
@@ -22,16 +26,17 @@ const SyntaxHighligter: FC<Props> = ({ children, lang }) => {
       });
 
       const code = highlighter.codeToHtml(children, {
-        lang: 'javascript',
+        lang,
         theme: 'github-dark',
+        transformers: [transformerNotationHighlight(), transformerNotationWordHighlight(), transformerNotationDiff()],
```

Comment:
Option to decorate code for clarity（for react）

Shikiji is used separately for React and Astro